db/db-challenge3/db-007/db-007.sql

```diff
@@ -0,0 +1,17 @@
+select
+  id,
+  chat_name,
+  chat_description,
+  created_by_user_id,
+  created_at,
+  updated_by_user_id,
+  updated_at,
+  case is_disabled_transmission
+    when 0 then '許可'
+    when 1 then '禁止'
+  end as is_disabled_transmission,
+  is_directchat, is_deleted
```

Comment:
せっかく改行しているので、このようにする感じが良いと思います。

```suggestion
  is_directchat, 
  is_deleted
```